integration-tests/pom.xml

```diff
@@ -190,14 +206,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>11</source>
```

Comment:
should we specify a java version or does it inherit this config?

Reply:
this indeed inherits